chukwuemekachm/Maintenance-Tracker
# #163355666 Completes setup of config files

#### What does this PR do?
- This PR completes the setup of configs and refactors chunks of existing code to use config.
#### Description of Task to be completed?
- Add a base env config module
- Add a migrations directory with seeding
- Add a global pool object
- Remove previous migration and seed handler files
- Modify database urls in controllers
- Setup prettier
#### How should this be manually tested?
- Setup the app [locally](https://github.com/chukwuemekachm/Maintenance-Tracker#installation)
- Create a test database and add it to the `.env` file
- Run `npm test`
- Observe that the tests are run on a test database
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#163355666](https://www.pivotaltracker.com/story/show/163355666)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
